### DISCUSSION TITLE
Folder structure

### DISCUSSION THREAD

**Discussion by tojorodialson** [livewire/livewire#1463] [4. General]
Hello guys!
Why with laravel livewire, all views are in folder called livewire ?
Why not and simply in ressources/views ?
Thank you

**Comment by nuernbergerA** [comment]
it's just a convention from livewire to put all livewire related view in a separate folder.
You can change this yourself if you like by just returning another view in your component `return view('what-ever-you-like.counter');`

**Reply by mblarsen** [reply]
Will you still be able to use `<livewire:what-ever-you-like.counter/>` or does that require configuring Livewire to look from the root views dir?

**Reply by nuernbergerA** [reply]
That's not affected by the `view`.

What you are talking about is the location of your `component`, which is `app/Http/Livewire/WhatEverYouLike/Counter.php`

**Reply by mblarsen** [reply]
I see, just being lazy here, I could have tried it out. Thanks for clearing that out.

**Reply by tojorodialson** [reply]
I find interesting use `<livewire:what-ever-you-like.counter/>`
